08_files_lab/01.py

```diff
@@ -1,6 +1,25 @@
-"""
-Write a program that takes 2 file names
-and appends the second file's contents to
-the end of the first one
-"""
-
+"""
+Elad Agam - Files
+01 - Get two files and add the content of the first to the second
+"""
+
+import argparse
+import os
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-s', '--source')
+parser.add_argument('-d', '--destination')
+args = vars(parser.parse_args())
+
+workingDir =  os.getcwd()
+
+source = os.path.join(workingDir, args['source'])
+destination = os.path.join(workingDir, args['destination'])
+
+
+with open(destination,"a") as fout:
+    with open(source,"r") as fin:
+        for line in fin:
+            fout.write(line)
+
+

```

Comment:
Appending the filename to the current directory is not necessary and actually might be even problematic if the user gives absoulte path from the root of the filesystem. Like this

```python 01.py /tmp/a  /tmp/b```